strawberryfields/apps/qchem/utils.py

```diff
@@ -96,3 +96,59 @@ def duschinsky(
     delta = np.array(d @ l0_inv)
 
     return U, delta
+
+
+def read_gamess(file) -> Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
+    r"""Reads molecular data from the output file generated by the GAMESS quantum chemistry package.
+
+    This function extracts the atomic coordinates, atomic masses, vibrational frequencies,
+    and normal modes of a molecule from the output file of a vibrational frequency calculation
+    performed with the GAMESS quantum chemistry package.
+
+    **Example usage:**
+
+    >>> r, m, w, l = read_gamess('../BH_frq.out')
+    >>> r
+    array([0.000000, 0.000000, 0.000000, 1.253604, 0.000000, 0.000000]
+
+    Args:
+        file: name and the path of the GAMESS output file
```

Comment:
Maybe just say "path to the GAMESS output file" since the path includes the name?

Reply:
You may add a keyword argument so the user can indicate the path to his/her file like we do in the function ``read_structure`` of PL-QChem. https://github.com/PennyLaneAI/pennylane/blob/936a5cf626b2f933dbb6a766bb1baa78a739cfcc/qchem/pennylane_qchem/qchem/structure.py#L149